demos/tests/cases.py

```diff
@@ -68,6 +70,33 @@ def fixed_args(self, source_dir, build_dir):
             *(['-l', str(cpu_extension_path)] if cpu_extension_path.exists() else [])]
 
 
+class NotebookDemo(Demo):
+    """
+    Convert notebook to Python script and runs it
+    """
+    def __init__(self, name, implementation='jupyter-python', device_keys=None, test_cases=None):
+        super().__init__(name, implementation, device_keys, test_cases)
+        self._exec_name = self._exec_name.replace('_jupyter-python', '')
+
+    def fixed_args(self, source_dir, build_dir):
+        sys.path.append(os.path.join(os.path.dirname(os.path.abspath(os.curdir)), "common", "python"))
```

Comment:
```suggestion
        demo_env = {**os.environ,
                   'PATH': f"{os.environ['PATH']}{os.pathsep}"
                           f"{os.path.dirname(sys.executable)}",
                   'PYTHONPATH': f"{os.environ['PYTHONPATH']}{os.pathsep}"
                                 f"{os.path.join(os.path.dirname(os.path.abspath(os.curdir)), 'common', 'python')}"}
```
Possible fix for `FileNotFoundError: [Errno 2] No such file or directory: 'jupyter': 'jupyter'`